src/Entity/ListBuilder/SubscriptionListBuilder.php

```diff
@@ -210,13 +184,13 @@ public function buildRow(EntityInterface $entity) {
           'data' => Link::fromTextAndUrl($rate_plan->getDisplayName(), $this->ratePlanUrl($entity)),
           'class' => ['rate-plan-name'],
         ],
-        'package' => [
-          'data' => $rate_plan->getPackage()->getDisplayName(),
-          'class' => ['package-name'],
-        ],
-        'products' => [
-          'data' => $products,
-          'class' => ['products'],
+        'status' => [
+          'data' => [
+            '#type' => 'inline_template',
+            '#template' => '<span class="status status--{{ status|clean_class }}">{{ status }}</span>',
+            '#context' => ['status' => $this->t('@status', ['@status' => $entity->getSubscriptionStatus()])],
+          ],
+          'class' => ['field-status'],
```

Comment:
While we are here, if you don't mind, could we try to make these consistent?  This one (not added by you) seems especially problematic (too general):
 
<img width="383" alt="Screen Shot 2019-06-07 at 2 01 56 PM" src="https://user-images.githubusercontent.com/60979/59124447-9f42ce80-892d-11e9-8ac1-258a37b58b7c.png">

Maybe...

- `.field-status` => `.subscription-status`
- `.rate-plan-name` => `.subscription-name` or `.subscription-plan`

Open to better suggestions too ;)

Reply:
Done. I've updated the class names.

![Purchased_plans___Drupal_8_m10n](https://user-images.githubusercontent.com/124599/59175358-4e69ea80-8b66-11e9-9fd5-ce351ae3c021.jpg)